cmd/queue/main.go

```diff
@@ -198,6 +205,10 @@ func probeUserContainer() bool {
 // Make handler a closure for testing.
 func handler(reqChan chan queue.ReqEvent, breaker *queue.Breaker, handler http.Handler) func(http.ResponseWriter, *http.Request) {
 	return func(w http.ResponseWriter, r *http.Request) {
+		recorder := pkghttp.NewResponseRecorder(w, http.StatusOK)
```

Comment:
Why do we need a response recorder here? We're not using any metadata generated by it. In it's current form, I don't think it's needed.

Reply:
You're correct, it's not being used.